canvas_account_admin_tools/templates/canvas_account_admin_tools/dashboard_account.html

```diff
@@ -4,68 +4,75 @@
 
 {% block stylesheets %}
 <link href="{% static 'canvas_account_admin_tools/css/dashboard.css' %}" rel="stylesheet" />
+  <style>
+    a.btn {
+        width: 20em;
+        margin-top: 1em;
+    }
+  </style>
 {% endblock stylesheets %}
 
 {% block content %}
-<header>
-    <div class="row lti-header">
-        <h1>Administration Tasks</h1>
-    </div>
-</header>
 
-<main>
+    <header>
+      <nav class="ng-scope">
+        <h3>Admin Console</h3>
+      </nav>
+    </header>
+
     {% if has_manage_courses %}
-    <div class="col-md-6">
-        <div class="panel panel-default">
-            <div class="panel-body">
-                <h3>Manage Courses</h3>
-                <div class="list-group">
-                    {% if canvas_site_creator %}
-                    <a href="{{ canvas_site_creator }}" class="list-group-item active" target="_parent">
-                        <h4 class="list-group-item-heading">Canvas Site Creator</h4>
-                        <p class="list-group-item-text">Create Canvas sites for one or multiple courses</p>
-                    </a>
-                    {% endif %}
-                </div>
-            </div>
+
+      {% if canvas_site_creator %}
+        <div class="btn-group col-md-4">
+          <a href="{{ canvas_site_creator }}" class="list-group-item active" target="_parent">
+              <h4 class="list-group-item-heading">Canvas Site Creator</h4>
```

Comment:
If these are `list-group-item`s, then the wrapping div probably wants to be `list-group`, not `btn-group`, right?  Especially since there's no buttons, or anything with `class="btn"`, in it?